exercicios/para-casa/sistema_de_alunas_mariana_araujo.py

```diff
@@ -0,0 +1,157 @@
+from dataset_alunas import dataset
+
+def main():
+    print("\n---  Seja bem vinda a Escola do Reprograma!  ---")
+    print("Sistema de informações de alunas")
+    
+    while True:
+        cod_opcao = obter_opcao()
+        
+        if cod_opcao == 1: incluir_nova_aluna()
+        elif cod_opcao == 2: consultar_lista_alunas()
+        elif cod_opcao == 3: consultar_faltas_aluna()
+        elif cod_opcao == 4: consultar_notas_aluna()
+        elif cod_opcao == 5: consultar_status_aprovacao()
+        elif cod_opcao == 6: print("Encerrando o programa..."); break
+
+def obter_opcao():
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+        return codigo_opcao
+    
+def incluir_nova_aluna():
+    print('\nInsira os seguintes dados: ')
+    nome = input('Nome da aluna: ')
+    sobrenome = input('Sobrenome da aluna: ')
+    turma = input('Turma da aluna: ') 
+    lista_de_presenca = presenca(nome, sobrenome) 
+    nota_participacao = float(input('Nota de participação da aluna (0-10): ')) 
+    notas = obter_notas(nome,sobrenome)
+
+    salvar_dados_aluna(nome, sobrenome, turma, notas, lista_de_presenca, nota_participacao)
+    return incluir_nova_aluna
+
+
+def presenca(nome,sobrenome):
```

Comment:
O nome da função poderia ser mais descritivo para indicar o que ela faz, pense sobre, o que ela faz com a presença? Ela poderia recuperar a presença, incluir presença, alterar a presença, etc..

Outro ponto, é que recebemos os argumentos nome e sobrenome, mas não estamos utilizando na função. Então podemos remover neste ponto pra que não tenha complexidade desnecessária.

**Sugestão:**
```suggestion
def obter_presenca_aluna():
```